team-fingerprints-web/src/components/FiltersSets/FiltersSets.tsx

```diff
@@ -188,29 +204,30 @@ const FiltersSets = ({
 
   return (
     <div className="filters">
-      {isPublic && (
-        <div className="filters__item">
-          <div className="filters__icon">
-            <ColoredShape shape="circle" color="#32A89C" />
-          </div>
-          <span className="filters__name filters__name--cursor-default">
-            Global Results
-          </span>
+      <div className="filters__item">
+        <div className="filters__icon">
+          <ColoredShape shape="circle" color="#32A89C" />
         </div>
+        <span className="filters__name filters__name--cursor-default">
+          {copyForDefaultResults}
+        </span>
+      </div>
+      {Object.values<FiltersSet>(filterSets)?.map(
+        (filterSet) =>
+          !filterSet?.user && (
+            <ResultsFilters
```

Comment:
How about filtering before mapping? I think it would look cleaner 🤔 
```suggestion
      {Object.values<FiltersSet>(filterSets)
        ?.filter((filterSet) => !filterSet?.user)
        ?.map((filterSet) => (
          <ResultsFilters
```